KyberAdapter/contracts/interfaces/IKyberPool.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+interface IKyberPool {
+    function swap(
+        uint256 amount0Out,
+        uint256 amount1Out,
+        address to,
+        bytes calldata data
+    ) external;
+
+    function getTradeInfo()
+        external
+        view
+        returns (
+            uint112 _vReserve0,
+            uint112 _vReserve1,
+            uint112 reserve0,
+            uint112 reserve1,
+            uint256 feeInPrecision
+        );
+
+    function token0() external view returns (address);
```

Comment:
Missing NatSpec
Add descriptions for the interface, events and functions
Refer https://docs.soliditylang.org/en/v0.8.17/natspec-format.htm